From 7686ca3c821abd7dd23dcf39896343c233b6f458 Mon Sep 17 00:00:00 2001
From: Hien Dang <hien.dang.eb@rvc.renesas.com>
Date: Wed, 16 Dec 2015 17:44:54 +0700
Subject: [PATCH 1272/2066] thermal: rcar_gen3_thermal: Optimize the delay
 when waiting for irq

There is a small delay waiting for interrupt condition to be valid.
This patch will tune up this delay for faster interrupt response time.

Signed-off-by: Hien Dang <hien.dang.eb@rvc.renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 drivers/thermal/rcar_gen3_thermal.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/thermal/rcar_gen3_thermal.c b/drivers/thermal/rcar_gen3_thermal.c
index 7dcb895..d065531 100644
--- a/drivers/thermal/rcar_gen3_thermal.c
+++ b/drivers/thermal/rcar_gen3_thermal.c
@@ -129,7 +129,7 @@ static int rcar_gen3_thermal_update_temp(struct rcar_thermal_priv *priv)
 		} else
 			break;
 
-		udelay(300);
+		udelay(150);
 	}
 
 	priv->ctemp = round_temp(TEMP_CONVERT(ctemp));
-- 
1.7.5.4

